### PR TITLE
adds one use rpgs

### PR DESCRIPTION
**Pull request by Pariah919**
## About The Pull Request
i don't even know why I've done this, this code is awful

Anyways gameplay wise it adds an rpg that you can't unload, so you can't get ammo for it otherwise other than admins spawning it in for you. Making a one use RPG effectively.

## Why It's Good For The Game
It's cool and adds cool gameplay options if you sacrifice your belt slot or something

## Changelog
:cl:
add: Added one use rpg to the code
/:cl:

### CODE REVIEW

**Comment by tgstation-server** [comment]
You currently have a negative Fix/Feature pull request delta of -163. Maintainers may close this PR at will. Fixing issues or improving the codebase will improve this score.

**Comment by Kuro-ai** [comment]
literally insane

**Comment by BeloneX** [comment]
Add this to req for like 50 points.

**Comment by AtlasRules** [comment]
i keep da mf thang on me

the mf thang is a M72 LAW

**Comment by Pariah919** [comment]
this just needs real sprites i guess

File: code/modules/projectiles/guns/specialist.dm [hunk 1]
```diff
@@ -1036,6 +1036,40 @@
 	recoil = 3
 	scatter = -100
 
+//-------------------------------------------------------
+//M5 RPG
+
+/obj/item/weapon/gun/launcher/rocket/oneuse
+	name = "\improper T-72 rocket launcher"
+	desc = "This is the primier disposable rocket launcher used throughout the galaxy, isn't unloadable or reloadable on the field. This one a 68mm explosive rocket."
```
Comment (XSlayer300) [inline code/modules/projectiles/guns/specialist.dm:1044]:
```suggestion
	desc = "This is the premier disposable rocket launcher used throughout the galaxy, it cannot be reloaded or unloaded on the field. This one fires a 68mm explosive rocket."
```

**Review by TiviPlus — APPROVED** [review]

**Review by TiviPlus — APPROVED** [review]

**Comment by tgstation-server** [comment]
Your Fix/Feature pull request delta is currently below zero (-222). Maintainers may close future Feature/Tweak/Balance PRs. Fixing issues or helping to improve the codebase will raise this score.

**Pull request merged**